src/FixtureLoader/FixtureLoader.php

```diff
@@ -0,0 +1,54 @@
+<?php declare(strict_types=1);
+
+namespace Neusta\Pimcore\FixtureBundle\FixtureLoader;
```

Comment:
Why is this called `FixtureLoader` instead of just `Loader` (like `Executor`, `Locator`, or `Sorter`)?